src/query/v2/requests.hpp

```diff
@@ -450,10 +450,12 @@ struct ExpandOneResponse {
   std::vector<ExpandOneResultRow> result;
 };
 
-struct UpdateVertexProp {
+struct UpdateVertex {
   PrimaryKey primary_key;
   // This should be a map
-  std::vector<std::pair<PropertyId, Value>> property_updates;
+  std::vector<LabelId> add_labels;
+  std::vector<LabelId> remove_labels;
```

Comment:
Please put a comment about the order of adding and removing labels, which one happens first. I think the order doesn't matter right now, so it okay whatever order you implement, but I know for sure it will cause some confusion at some point without documenting it.